### DISCUSSION TITLE
Out dated of mobile android device model vivo 1906 but updated it into android 11 version and would like to using with latest of updated android apps, google chrome, chrome browser version etc.,

### DISCUSSION THREAD

**Discussion by Yee2266156** [community/community#78346] [Mobile]
### Select Topic Area

Product Feedback

### Body

Out dated of mobile android device model vivo 1906 but updated it into android 11 version and would like to using with latest of updated android apps, google chrome, chrome browser version etc.,

**Comment by stevepopovich** [comment]
Hi @Yee2266156 this forum is feedback about the GitHub Mobile app. For general Android tech help, I would try general Google searching, or talking to your mobile provider
